source/data/__tests__/data-test.js

```diff
@@ -44,6 +44,21 @@ describe('data sending module', () => {
     ))
   })
 
+  it('should call jeffrey.trackAction with a properly formatted submit feedback event', () => {
+    sendNPSFeedback({ pageId: 'test-page', userId: 'test-user', score: 3, feedback: 'Love you guys!' })
+
+    assert(jeffrey.trackAction.called)
+    assert(jeffrey.trackAction.calledWith(
```

Comment:
Is `called` followed by `calledWith` not redundant? (as in if it's "called with", it must have been called?)